modules/representation_management/app/controllers/representation_management/v0/pdf_generator_2122_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    class PdfGenerator2122Controller < RepresentationManagement::V0::PdfGeneratorBaseController
+      # service_tag 'lighthouse-veteran' # Is this the correct service tag?
+
+      def create
+        # We'll need a process here to check the params to make sure all the
+        # required fields are present. If not, we'll need to return an error
+        # with status: :unprocessable_entity.  If all fields are accounted for
+        # we need to fill out the 2122 PDF with the data and return the file
+        # to the front end.
+
+        # This work probably belongs in the PDF Generation ticket.
+        render json: {}, status: :unprocessable_entity
+      end
+
+      private
+
+      def form_params
+        params.permit(all_params)
+      end
+
+      def all_params
+        [
+          claimant_params,
+          service_organization_params,
+          veteran_params,
+          :record_consent,
+          :consent_address_change,
+          { consent_limits: [] }
+        ].flatten
+      end
+
+      def service_organization_params
+        %i[
+          service_organization_name
+          service_organization_representative_name
+          service_organization_job_title
+          service_organization_email
+          service_organization_appointment_date
+
+        ]
+      end
+
+      def feature_enabled
```

Comment:
This method can be deleted - it's inherited from `PdfGeneratorBaseController`

Reply:
Done.